docs/examples/demo-object-as-source.html

```diff
@@ -24,7 +24,7 @@
   <p>Selected: <code>{{ctrl.person.selectedSingle}}</code></p>
   <ui-select ng-model="ctrl.person.selectedSingle" theme="select2" ng-disabled="ctrl.disabled" style="min-width: 300px;" title="Choose a person">
     <ui-select-match placeholder="Select a person in the list or search his name/age...">{{$select.selected.value.name}}</ui-select-match>
-    <ui-select-choices repeat="person.value.name as (key, person) in ctrl.peopleObj | filter: { value : { name: $select.search }}">
+    <ui-select-choices repeat="person.value.name as (key, person) in ctrl.peopleObj | filter: $select.search">
```

Comment:
`filter: { value : { name: $select.search }}` was used here as I believe `filter: $select.search` does not work correctly. (It might angular v1.5+ only).